content/en/metrics/custom_metrics/historical_metrics.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Historical Metrics Ingestion
+kind: documentation
+further_reading:
+- link: "/developers/dogstatsd/"
+  tag: "Documentation"
+  text: "Learn more about DogStatsD"
+- link: "/developers/community/libraries/"
+  tag: "Documentation"
+  text: "Official and Community created API and DogStatsD client libraries"
+---
+
+## Overview
+
+Historical Metrics Ingestion allows you to collect metric points with outdated timestamps. Timestamps that are older than one hour from the time of submission, but no older than your total metric retention period (which defaults to 15 months for all metrics).
+
+With Historical Metrics Ingestion, you can monitor a variety of new use cases with your metrics, such as outage recovery, overwriting invalid metrics, and managing IoT delays.
+
+## What is Historical Metric Ingestion?
+
+{{< img src="metrics/custom_metrics/historical_metrics/diagram_historical-metrics-ingestion_intro.png" alt="Diagram showing the ingestion flow for Historical Metrics" >}}
+
+Datadog classifies *historical metrics* as metric points with timestamps that are older than an hour relative to the time of submission. 
```

Comment:
Metrics with historical metric ingestion enabled in Datadog can have values with timestamps that are _older_ than one hour relative to the time those values were submitted. Without historical metric ingestion enabled, the timestamps on your metric values' timestamps must be < one hour ago relative to the time those values are submitted.

Reply:
```suggestion
Datadog classifies *historical metrics* as metric points with timestamps that are older than an hour relative to the time of submission. If Historical Metrics Ingestion is not enabled, metrics older than an hour from submission will not be ingested.
```